opm/simulators/utils/readDeck.cpp

```diff
@@ -324,9 +324,16 @@ void readDeck(int rank, std::string& deckFilename, std::unique_ptr<Opm::Deck>& d
             eclipseState = std::make_unique<Opm::ParallelEclipseState>();
     }
 
+    // In case of parse erorors eclipseState/schedule might be null
```

Comment:
Tiny nit: `erorors` should be `errors`.  Otherwise this looks good to me.

Reply:
Sigh, my typing just sucks. Maybe this 4.5 finger system of mine does not work :smile: 
Done and force-pushed.